openstack/neutron/templates/vct-nsxv3-agent-deployment.yaml

```diff
@@ -43,6 +43,7 @@ template: |
           configmap-etc-hash: {{ include (print $.Template.BasePath "/configmap-etc.yaml") . | sha256sum }}
           prometheus.io/scrape: "true"
           prometheus.io/targets: {{ required ".Values.metrics.prometheus missing" .Values.metrics.prometheus | quote }}
+          {{- include "utils.linkerd.pod_and_service_annotation" $ | indent 10 }}
```

Comment:
Just wondering: did it work properly with the `$` here? The other includes use `.` in this file.

Reply:
It was failing when I tried . so I used $

Reply:
I tested it and it works with . but I left $ for consistency, since on some other places . is failing. Do you think it would be better to use . or leave $ everywhere ?

Reply:
I'd try to be consistent with the usage of `.` and `$` within the same file.

Reply:
OK makes sense, updated it as well

Reply:
I just marked this one, but there are lots of others, too, that use `$` even though the rest of the file uses `.`. Since you decided to be consistent in a file, could you please do the same in the other files, too?